modin/pandas/base.py

```diff
@@ -3630,7 +3630,9 @@ def value_counts(
     ):
         if subset is None:
             subset = self._query_compiler.columns
-        counted_values = self.groupby(by=subset, dropna=dropna, observed=True).size()
+        counted_values = self.groupby(
+            by=subset, dropna=dropna, observed=True, sort=False
+        ).size()
```

Comment:
groupby on Ray always raises this warning with `sort=False`:
https://github.com/modin-project/modin/blob/f14b0fe6db4755903cd52ddd3b51e4aff172c359/modin/core/dataframe/algebra/groupby.py#L381-L387

Can you please wrap this groupby call into something like this in order to silence the warning?
```suggestion
        with warnings.catch_warnings():
            warnings.filterwarnings(
                "ignore",
                message=".*groupby keys will be sorted anyway.*",
                category=UserWarning,
            )
            counted_values = self.groupby(
                by=subset, dropna=dropna, observed=True, sort=False
            ).size()
```